class.jetpack-tracks.php

```diff
@@ -13,6 +13,10 @@ static function track_jetpack_usage() {
 			return;
 		}
 
+		if ( ! defined( 'DOING_AJAX' ) && ! Jetpack::is_user_connected( $user_id = get_current_user_id() ) ) {
```

Comment:
Can you explain why this change is necessary?

Reply:
See https://github.com/Automattic/jetpack/pull/7620#discussion_r135115869